README.md

```diff
@@ -1 +1,7 @@
-# inventor-be
\ No newline at end of file
+# inventor-be
+
+# Link to E R Diagram
+[SoftwareIntern: Inventor](https://online.visual-paradigm.com/share.jsp?id=323232313032392d35#diagram:workspace=jllhkccg&proj=0&id=5)
+
+![Entity Relationship Diagram](![SoftwareIntern_Inventor](https://github.com/cypherkrescent/inventor-be/assets/78306383/a17fba1b-ff38-4367-8cbf-dd3ae9f28d64)
```

Comment:
- The image isn't displayed 
- Open one pr for a issue
- Attach this pr to issue #1 . check out [Closing Keywords](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue)

@Daveohi